src/VideoCommand.cc

```diff
@@ -0,0 +1,352 @@
+/**
+ * @file   VideoCommand.cc
+ *
+ * @section LICENSE
+ *
+ * The MIT License
+ *
+ * @copyright Copyright (c) 2017 Intel Corporation
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction,
+ * including without limitation the rights to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+ * ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ */
+
+#include <iostream>
+#include <fstream>
+
+#include "VideoCommand.h"
+#include "VDMSConfig.h"
+#include "defines.h"
+
+using namespace VDMS;
+
+VideoCommand::VideoCommand(const std::string &cmd_name):
+    RSCommand(cmd_name)
+{
+}
+
+void VideoCommand::enqueue_operations(VCL::Video& video, const Json::Value& ops)
+{
+    // Correct operation type and parameters are guaranteed at this point
+    for (auto& op : ops) {
+        const std::string& type = get_value<std::string>(op, "type");
+         std::string unit ;
+        if (type == "threshold") {
+            video.threshold(get_value<int>(op, "value"));
+
+        }
+        else if (type == "interval") {
+
+            video.interval(
+                VCL::Video::FRAMES,
+                get_value<int>(op, "start"),
+                get_value<int>(op, "stop"),
+                get_value<int>(op, "step"));
+
+        }
+        else if (type == "resize") {
+             video.resize(get_value<int>(op, "height"),
+                           get_value<int>(op, "width") );
```

Comment:
w and h on wiki. Also, I think the operations could be explained a little more in the context of video on the wiki. + an example of how a blob could be specified.

Reply:
Cool, glad you realized. Would you update the wiki? you have full access. 
I have just made sure that it is consistent with the JSON schema, so you can use the code as reference for updating the wiki.

Reply:
I actually prefer the w, h etc that's on the wiki (minus some formatting issues there). Mapping it to current code will make it inconsistent with what bounding box API has. Christina and I had discussed the variable names and she had made them match in code and wiki for that commit.

Reply:
I see. Doing that for the boundingbox was a mistake, as it broke consistency with what we already had for operations (shame we didn't catch that before). 
Image operations use (and have always used) height and width. Videos use the same operations block.
We should keep them consistent with the images as they are now.

Reply:
I dont think it was a mistake. A rectangle block with x, y, w, h makes sense. With crop for example, keeping x, y and then width and height is looking weird now.